tests/src/sso-extension.spec.ts

```diff
@@ -0,0 +1,164 @@
+let extensionInstalled = false;
+const imageName = 'ghcr.io/redhat-developer/podman-desktop-redhat-account-ext:latest';
+const extensionName = 'Red Hat Authentication';
+const extensionLabelName = 'redhat-authentication';
+const authProviderName = 'Red Hat SSO';
+const extensionStatusLabel = 'Extension Status Label';
+const extensionDetailsLabel = 'Extension Details';
+const activeExtensionStatus = 'ACTIVE';
+const disabledExtensionStatus = 'DISABLED';
+
+
+beforeEach<RunnerTestContext>(async ctx => {
+  ctx.pdRunner = pdRunner;
+});
+
+beforeAll(async () => {
+  pdRunner = new PodmanDesktopRunner();
+  page = await pdRunner.start();
+  pdRunner.setVideoAndTraceName('sso-e2e');
+
+  const welcomePage = new WelcomePage(page);
+  await welcomePage.handleWelcomePage(true);
+  navBar = new NavigationBar(page);
+});
+
+afterAll(async () => {
+  await pdRunner.close();
+});
+
+describe('Red Hat Authentication extension verification', async () => {
+
+  test('Go to settings and check if extension is already installed', async () => {
+    const settingsBar = await navBar.openSettings();
+    const extensions = await settingsBar.getCurrentExtensions();
+    if (await extensionExists(extensions, extensionName)) {
+      extensionInstalled = true;
+    }
+  });
+
+  test.runIf(extensionInstalled)(
+    'Uninstalled previous version of sso extension',
+    async () => {
+      await removeExtension(extensionName);
+    },
+    60000,
+  );
+
+  test('Extension can be installed using OCI image', async () => {
+    const settingsBar = await navBar.openSettings();
+    const settingsExtensionPage = await settingsBar.openTabPage(SettingsExtensionsPage);
+    await playExpect(settingsExtensionPage.heading).toBeVisible();
+    await settingsExtensionPage.installExtensionFromOCIImage(imageName);
+  }, 200000);
+
+  test('Extension record is added under installed extensions', async () => {
+    const settingsBar = await navBar.openSettings();
+    const settingsExtensionPage = await settingsBar.openTabPage(SettingsExtensionsPage);
+    const extension = settingsExtensionPage.getExtensionRowFromTable(extensionLabelName);
+    await playExpect(extension).toBeVisible();
+    await extension.scrollIntoViewIfNeeded();
+    const extensionDetails = extension.getByRole('cell', { name: extensionDetailsLabel });
+    console.log(`details text: ${await extensionDetails.allInnerTexts()}`);
+    await playExpect(extensionDetails.getByLabel(extensionStatusLabel)).toHaveText(activeExtensionStatus);
+  });
+
+  test('Extension appears under Settings bar extensions', async () => {
+    const settingsBar = await navBar.openSettings();
+    const extensions = await settingsBar.getCurrentExtensions();
+    await playExpect.poll(async () => await extensionExists(extensions, extensionName), { timeout: 30000 }).toBeTruthy();
+  });
+
+  test('SSO provider is available in Authentication Page', async (context) => {
+    console.log(`Running Test: ${context.task.name}`);
```

Comment:
Are those console.log still required or should they be removed ?

Reply:
Unnecessary. Will remove.